components/form/FormItem.tsx

```diff
@@ -26,6 +25,14 @@ export interface FormItemProps {
   colon?: boolean;
 }
 
+function intersperseSpace<T>(list?: Array<T>): Array<T | string> {
+  return (list || [])
```

Comment:
if `list` is a empty array, slice(1) will cause a error.
`````
if (list.length) return [];
return list.reduce((current, item) => (
      [...current, ' ', item]
    ), [])
    .slice(1);
``````

Reply:
Sorry. I miss it. Do you mean `[].slice(1)` cause error?

Reply:
Sorry for my bad. If index out of range, slice will return a empty array. Just test for now.
``````
[].slice(1)
[]
``````